dogpile/cache/backends/memcached.py

```diff
@@ -281,17 +279,16 @@ class MemcacheArgs(GenericMemcachedBackend):
     """
 
     def __init__(self, arguments: MemcachedArgsArguments):
-        _arguments = cast(Dict, arguments.copy())
-        self.min_compress_len = _arguments.get("min_compress_len", 0)
+        self.min_compress_len = arguments.get("min_compress_len", 0)
 
         self.set_arguments = {}
-        if "memcached_expire_time" in _arguments:
-            self.set_arguments["time"] = _arguments["memcached_expire_time"]
-        if "min_compress_len" in _arguments:
-            self.set_arguments["min_compress_len"] = _arguments[
+        if "memcached_expire_time" in arguments:
+            self.set_arguments["time"] = arguments["memcached_expire_time"]
+        if "min_compress_len" in arguments:
+            self.set_arguments["min_compress_len"] = arguments[
                 "min_compress_len"
             ]
-        _arguments_super = cast(GenericMemcachedBackendArguments, _arguments)
+        _arguments_super = cast(GenericMemcachedBackendArguments, arguments)
```

Comment:
I believe you can now pass arguments without a cast, since `GenericMemcachedBackendArguments` is a superclass of `MemcachedArgsArguments`